services/git-service-impl/src/test/java/io/fabric8/launcher/service/git/AbstractGitServiceIT.java

```diff
@@ -476,16 +478,29 @@ private GitRepository createRepository(GitOrganization organization, String repo
         return repository;
     }
 
-    private GitRepository createRepository(String repositoryName, String description) {
+    protected GitRepository createRepository(String repositoryName, String description) {
         GitRepository repository = getGitService().createRepository(repositoryName, description);
         repositoriesToDelete.add(repository);
         return repository;
     }
 
-    private String generateRepositoryName(final int number) {
+    protected String generateRepositoryName(final int number) {
         return this.getClass().getSimpleName().toLowerCase() + "-" + testName.getMethodName().toLowerCase() + "-" + number;
     }
 
+    @Test
+    @Ignore
```

Comment:
Why `@Ignore`d?

Reply:
Because I use it when there is an error and repositories are not deleted (instead of cleaning manually). Maybe I should add a comment though.